specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -1,18 +1,26 @@
 
 # VEP-NNNN: Your short, descriptive title
 
-* **Author(s):** Paul Murphy (murphp15@tcd.ie),Iva Koleva (ikoleva@vmware.com)
+* **Author(s):** Paul Murphy (murphp15@tcd.ie), Iva Koleva (ikoleva@vmware.com), Dilyan Marinov (mdilyan@vmware.com)
 * **Status:** implementable
 
-- [Summary](#summary)
-- [Glossary](#glossary)
-- [Motivation](#motivation)
-- [Requirements and goals](#requirements-and-goals)
-- [High-level design](#high-level-design)
-- [API Design](#api-design)
-- [Detailed design](#detailed-design)
-- [Implementation stories](#implementation-stories)
-- [Alternatives](#alternatives)
+- [VEP-NNNN: Your short, descriptive title](#vep-nnnn-your-short-descriptive-title)
+  - [Summary](#summary)
+  - [Glossary](#glossary)
+  - [Motivation](#motivation)
+  - [Requirements and goals](#requirements-and-goals)
+    - [Goals:](#goals)
+    - [Non Goals:](#non-goals)
+  - [High-level design](#high-level-design)
+      - [Folder Structure](#folder-structure)
+      - [Package publishing](#package-publishing)
```

Comment:
nit: High-level design towards package publishing, would specify what packages we publish (one per shared-components and data-pipelines); details on where we publish those, seem detailed design release section

Reply:
I disagree. It would be a bit confusing to list that we publish the packages and not to immediately specify where we publish them.

Reply:
So then link to the according section, in favour of duplicating info. So in case of change, we prevent discrepancies.

Reply:
I'd much rather err on the side of making the documentation more readable than to prevent possible discrepancies if we change the registry in the future. This isn't code, after all.